sections/featured-blog.liquid

```diff
@@ -91,24 +91,46 @@
                 style="--animation-order: {{ forloop.index }};"
```

Comment:
Not related to this line or your PR. 
We're checking against the wrong setting on the wrapping `ul` element. We're using a conditional with `settings.card_style == 'standard'` which points to the product card settings. It should be `settings.blog_card_style == 'standard'`. 

Do you mind adding that to your PR ? It's a very small tweak 😅 Though usually we would want to do that in its own PR.

Reply:
I added this to my PR! It's such a small change, there's no reason to open a separate issue for it :D